.github/workflows/label-issue-open.yml

```diff
@@ -30,3 +41,27 @@ jobs:
         if: ${{ steps.latest-issue.outputs.hasLabels == 'false'}}
         with:
           labels: 'status: awaiting triage'
+
+  community:
+    name: Add community contribution label
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/github-script@v6
+
+      - name: Check if user is a community contributor
+        id: check
+        run: node .github/workflows/scripts/is-community-contributor.js
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It doesn't seem like we're using [the syntax documented for `actions/github-script`](https://github.com/actions/github-script#examples)